### ISSUE TITLE
data_utils.py line 45, where are the spec.pt files?

### ISSUE THREAD

**Issue by Penghmed** [OlaWod/FreeVC#22]
I dont have file like it, how to generate them? 
Get exception file not found.

**Comment by OlaWod** [comment]
those spec.pt files are generated on the fly during the first epoch of training.

**Comment by Penghmed** [comment]
Had to run single GPU first thanks you

**Comment by skol101** [comment]
Yep, until all spect.pt files are generated, the training must be run on a single GPU
